ceph-dev-new-trigger/config/definitions/ceph-dev-new-trigger.yml

```diff
@@ -158,7 +158,7 @@
                   predefined-parameters: |
                     BRANCH=${GIT_BRANCH}
                     FORCE=True
-                    DISTROS=focal centos8
+                    DISTROS= centos8
```

Comment:
could you add a prefix for the title of the commit message? and please remove the space after "=".

Reply:
ack.